Milestone1

brugobi opened the pull request:
- set up files
- the linters is showing errors because I have no files that match, since this milestone 1 it is just setting up the files and deploy it: https://brugobi.github.io/Calculator-REACT/

thanks for the review

brugobi committed:
add linnters files

brugobi committed:
delete files from react

addod19 reviewed (changes requested):
## Status: Require changes :recycle: :traffic_light:

Hi @brugobi, great job on this project :+1: The following requirements have been met:
- App is initialized with create-react-app
- Big.js and prop-types has been added to the package.json file
- React SVG logo and webReportVitals file have also been deleted

The following requirements have not been met:
- The app does not run correctly, probably because there is not `src` directory, the index.js, app.js files have been deleted. Let us kindly add them because they are important for the running of the application :muscle: 
- Kindly add the link to the live version in our README file
- The project is missing an important requirement, which is the README should have a project description, instructions about how to clone and run the project locally, technologies used, social handles of the Author, etc
- Kindly fix the linter errors, we cannot approve the project without passing all the linter checks. This is a general requirement, it goes for all milestones

**NB**: We do not need to delete the src directory and all the files in them, let us delete only the files not needed for this project

[Optional]
- We have 2 lock files, this is as a result of using both `npm && yarn` in our installations. We need to use only 1 to avoid problems with the deployment


Once these are resolved, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

brugobi committed:
add App.js, index.js and css, reinstall linters and delete lock files

brugobi committed:
update README with info about the project and the usage

brugobi committed:
reset react files and delete unnecessary ones

brugobi committed:
add prop-types, big.js and linters files

brugobi committed:
update README with the app info

brugobi committed:
fix linters

brugobi committed:
add eslint json file and fix the linters issue

brugobi committed:
Create a React app and publish it to GitHub Pages

brugobi commented:
- I made all the changes as the TSE has suggested

- [X] added the src/ folder with the important files
- [X] fixed linters;
- [X] deployed
- [X] added README with the information about the app included the link to the live version; 
- [X] deleted lock files

### thank you for the review

Srjnnnn reviewed (approved):
## Status: Approved :partying_face: 

Hello @brugobi. Thanks for making all the requested changes. You are good to go. :100: 
You can merge and move on to the next milestones.

_Eyüp Sercan UYGUR_
